lib/cogctl/actions/bundle/install.ex

```diff
@@ -80,18 +81,28 @@ defmodule Cogctl.Actions.Bundle.Install do
   end
 
   defp parse_config(params) do
-    with {:ok, config}         <- parse_string_or_file(params.file),
+    with {:ok, config}         <- parse_bundle_or_path(params.bundle_or_path),
          {:ok, templates}      <- build_template_map(params.templates),
          {:ok, amended_config} <- maybe_add_templates(templates, config),
          {:ok, fixed_config}   <- validate_config(amended_config),
          do: {:ok, fixed_config}
   end
 
-  defp parse_string_or_file(file) do
-    if File.exists?(file) do
-      Spanner.Config.Parser.read_from_file(file)
-    else
-      Spanner.Config.Parser.read_from_string(file)
+  defp parse_bundle_or_path(bundle_or_path) do
+    cond do
+      File.exists?(bundle_or_path) ->
+        Spanner.Config.Parser.read_from_file(bundle_or_path)
+      match?({:ok, _}, Poison.decode(bundle_or_path)) ->
```

Comment:
I think I missed how standard input gets piped down here... Also, that usecase doesn't seem to be described in the doc strings above.